dataland-batch-manager/src/test/kotlin/org/dataland/batchmanager/service/GleifApiAccessorTest.kt

```diff
@@ -0,0 +1,42 @@
+package org.dataland.batchmanager.service
+
+import org.apache.commons.io.FileUtils
+import org.dataland.datalandbatchmanager.service.GleifApiAccessor
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.assertThrows
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.mockStatic
+import org.mockito.Mockito.`when`
+import java.io.File
+import java.io.FileNotFoundException
+import java.net.SocketException
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class GleifApiAccessorTest {
+    @BeforeAll
+    fun setup() {
+        mockStatic(FileUtils::class.java)
+    }
+
+    @Test
+    fun `test download failure`() {
+        `when`(FileUtils.copyURLToFile(any(), any())).thenThrow(SocketException("Test Exception"))
+        assertThrows<FileNotFoundException> {
+            GleifApiAccessor().getFullGoldenCopy(File("test"))
+        }
+    }
```

Comment:
Man könnte hier noch die retries zählen und asserten. Ähnlich wie beim unit Test des CompanyUploaders

Reply:
Das ist ja implizit gecheckt, da die getestete Exception in genau dem Fall geworfen wird. Habs dennoch hinzugefügt.